Make sure that the vendor pattern for installation is passed through to
MakeMaker.  This is not suitable for upstream.

--- git-2.4.5/perl/Makefile	Thu Jun 25 12:15:40 2015
+++ git-2.4.5/perl/Makefile	Sat Jul  4 11:32:21 2015
@@ -80,7 +80,7 @@
 	echo '	echo $(instdir_SQ)' >> $@
 else
 $(makfile): Makefile.PL ../GIT-CFLAGS
-	$(PERL_PATH) $< PREFIX='$(prefix_SQ)' INSTALL_BASE='' --localedir='$(localedir_SQ)'
+	$(PERL_PATH) $< PREFIX='$(prefix_SQ)' INSTALL_BASE='' --localedir='$(localedir_SQ)' INSTALLDIRS=vendor
 endif
 
 # this is just added comfort for calling make directly in perl dir
